spark/src/main/java/org/apache/iceberg/spark/actions/BaseRemoveOrphanFilesSparkAction.java

```diff
@@ -140,10 +142,19 @@ public BaseRemoveOrphanFilesSparkAction deleteWith(Consumer<String> newDeleteFun
 
   @Override
   public RemoveOrphanFiles.Result execute() {
-    JobGroupInfo info = newJobGroupInfo("REMOVE-ORPHAN-FILES", "REMOVE-ORPHAN-FILES");
+    JobGroupInfo info = newJobGroupInfo("REMOVE-ORPHAN-FILES", getDescription());
     return withJobGroupInfo(info, this::doExecute);
   }
 
+  private String getDescription() {
+    List<String> msg = new ArrayList<>();
+    msg.add("older_than=" + olderThanTimestamp);
+    if (location != null) {
+      msg.add("location=" + location);
+    }
+    return String.format("Removing orphan files(%s) from %s", StringUtils.join(msg, ","), table.name());
```

Comment:
We also need to add dry_run to the description.

Reply:
I can see that this option is available in org.apache.iceberg.spark.procedures.RemoveOrphanFilesProcedure but not as part of the actions API. Please let me know if I am missing something

Reply:
Oh, yeah, we handle it by passing an empty delete function. Never mind.